sources/packages/web/src/store/modules/student/student.ts

```diff
@@ -0,0 +1,53 @@
+
+export class StudentState {
+
+     profile!: StudentProfile;
+}
+
+export class StudentProfile {
+    name?: string;
+    age?: string;
+    birthdate?: string;
+    displayName?: string;
+    email?: string;
+    emailVerified?: string;
+    familyName?: string;
+    gender?: string;
+    givenName?: string; 
+    middleName?:string;
+}
+
+const initialState: StudentState = {
+    profile: new StudentProfile()
+}
+
+export const student = {
+    namespaced: true,
+    state : initialState,
+   
+    mutations: {
```

Comment:
Is there any particular reason to keep the store's components (e.g. mutations, actions, etc.) in the same file instead keep them in separate files? IMHO, as they grow, it would favor maintenance and organization, but I am open to what the team decided on this matter.

Reply:
Sure. Usually i don't like having so many small files. if and when things do grow and this needs to be broken, we can. Also, it will help will need to have one definition of whats long enough to be broken

Reply:
Kept it as is as decided, this can be done later when there are more mutations